pages/index.tsx

```diff
@@ -182,11 +182,12 @@ export const HomePage: NextPage = () => {
       <Stack
         bgColor="Primary.50"
         py="72px"
-        px="112px"
+        px="10%"
         direction={{ base: 'column', md: 'row' }}
         align={{ base: undefined, md: 'center' }}
         spacing="16px"
         justify="space-between"
+        minWidth="800px"
```

Comment:
The search bar already took up the full-width of the page - padding without this?